### ISSUE TITLE
[SyntaxError: JSON Parse error: Unexpected character: '] when ran on iOS simulator inside react-native app

### ISSUE THREAD

**Issue by silencer07** [damonwonghv/youtube-search-api#28]
works fine with android.

**Comment by AdamElitzur** [comment]
@silencer07 I got the same issue, did you find a solution or alternative for react-native? I'm using expo, and would love for this to work! Thanks!

**Comment by silencer07** [comment]
Hi @AdamElitzur,

You will need to basically need to have an ios version of the code.

- copy paste the code
- reverse engineer it one by one to make it work, the error is due to the fact that for some reason when running in ios, the returned values are different

Or maybe you can try experiment and maybe find a way to force this library to send a header similar to what android is sending. Maybe that would make the response of server similar to android

**Comment by AdamElitzur** [comment]
Hi @silencer07, thanks so much. Looks very complicated, but I'll look into it! Did you fork the repo and make a new npm package, or what did you do?

**Comment by silencer07** [comment]
@AdamElitzur  I copy pasted the code and created my own js file and just modified what I did.

I did not publish npm because it is for hobby app and the code is dirty :D

Good luck!

**Comment by AdamElitzur** [comment]
Ok, sounds good, thank you!

**Comment by AdamElitzur** [comment]
Oh one thing @silencer07, were you able to publish an app with this to the app store? I want to make sure it'll get approved. Thanks!

**Comment by silencer07** [comment]
Yep. Search for videoke party maker remote in both app store and play store

**Comment by collinscoder1** [comment]
> Yep. Search for videoke party maker remote in both app store and play store

Provide a link to play store i want to download the app.
